magda-web-client/src/Components/Search/SearchBox.scss

```diff
@@ -43,6 +44,7 @@
     }
     img{
       width: 40px;
+      height: auto;
```

Comment:
Tested on win7 / IE9
Search icon looked not in the correct w/h ratio:
![image](https://user-images.githubusercontent.com/674387/36576946-2f20c842-18a8-11e8-8924-28938ccb7836.png)
Change to `height:40px` will make it look better:
![image](https://user-images.githubusercontent.com/674387/36576977-53196254-18a8-11e8-8d71-75414858c049.png)